Makefile

```diff
@@ -69,38 +71,60 @@ test: unittest pep8
 
 ## Packaging
 
-package: cleanbuild
-	# PYDUNDASVER := $(shell $(PY3) -c 'from pydundas import __version__ as v; print(v)')
+package: cleanbuild getver
+	@echo Building Pydundas version $(PYDUNDASVER).
 	$(PY3) setup.py sdist bdist_wheel
+	@echo Built Pydundas version $(PYDUNDASVER).
+
+getver:
+PYDUNDASVER := $(shell $(PY3) -c 'from pydundas import __version__ as v; print(v)')
 
 ## Upload
 
+pypiversion: getver
+	@echo Module version: $(PYDUNDASVER).
+	@echo Latest version on test.pypi: $(shell curl -s 'https://test.pypi.org/pypi/pydundas/json' | jq -r '.info.version').
```

Comment:
You're making assumptions about the availability of curl and jq here. Don't really think it is a problem per se, but you might want to make it explicit.

Reply:
It's only for the owner of the pydundas pypi account, so it's OK indeed.